fs/procfs/fs_procfsproc.c

```diff
@@ -1297,13 +1303,28 @@ static ssize_t proc_groupfd(FAR struct proc_file_s *procfile,
           path[0] = '\0';
         }
 
+#if CONFIG_FS_BACKTRACE > 0
```

Comment:
move all backtrace logic into separate function and format backtrace after line 1300

Reply:
and add "BACKTRACE" label in column

```
FD  OFLAGS  TYPE POS       PATH
0   3       1    0         /dev/console 0x00000000004aa3e2
```

```
FD  OFLAGS  TYPE POS       PATH         BACKTRACE
0   3       1    0         /dev/console 0x00000000004aa3e2
```